docs/gitflow/gitflow.md

```diff
@@ -1,87 +1,108 @@
 # SN Gitflows
 
 Branches
-* *master*: permanent branch for production environments
-* *develop*: permanent branch for aggregating feature branches
-* *feature*: temporary branch for features and bugfixes
-* *staging*: temporary branch for each release, used for creating near production QA environment with released backend
-* *hotfix*: temporary branch for urgent and high importance fixes
+
+- _master_: permanent branch for production environments
+- _develop_: permanent branch for aggregating feature branches
+- _feature_: temporary branch for features and bugfixes
+- _staging_: temporary branch for each release, used for creating near production QA environment with released backend
```

Comment:
- _staging_: temporary branch for each release, used for creating near production QA environment with released backend
```suggestion
- _release_: temporary branch for each release, used for creating near production QA environment with released backend
```